### ISSUE TITLE
TypeError: undefined is not a function

### ISSUE THREAD

**Issue by nanjusoil** [e14n/pump.io#1141]
I installed as taught in the ubuntu version guide.
when i type localhost it worked
but when i type localhost/:username
the browser keep showing this error and doesnt disappear even if I type other urls
but changing browser solves it
but still same problem after localhost/:username

`TypeError: undefined is not a function
    at /srv/pump.io/lib/app.js:344:37
    at Function._.each._.forEach (/srv/pump.io/node_modules/underscore/underscore.js:158:9)
    at defang (/srv/pump.io/lib/app.js:342:19)
    at /srv/pump.io/lib/app.js:348:37
    at Function._.each._.forEach (/srv/pump.io/node_modules/underscore/underscore.js:158:9)
    at Object.defang (/srv/pump.io/lib/app.js:342:19)
    at Object.eval (eval at <anonymous> (/srv/pump.io/node_modules/underscore/underscore.js:1454:20), <anonymous>:99:24)
    at Object.template (/srv/pump.io/node_modules/underscore/underscore.js:1461:21)
    at ServerResponse.res._render (/srv/pump.io/node_modules/express/lib/view.js:425:21)
    at ServerResponse.res.render (/srv/pump.io/node_modules/express/lib/view.js:318:17)`

**Comment by blog2read** [comment]
I get exactly same "white page" with this error message in the situation, when I'am clicking the button "back" in the browser.

**Comment by profOnno** [comment]
If I'm correct there is a problem in the git version at the moment. Use `npm install -g pump.io
` to install instead of the git command (`-g` is for global installation). 

Or try this version (I'm not sure if it's the same issue): `git clone --branch defang https://github.com/profOnno/pump.io.git`. 

The pull request is still pending. You can add a comment to it, if its working for you. #1144

**Comment by bastlwasti** [comment]
I got debian stable, and compiled pumpio.git yesterday - it worked after some hassles and compile errors. Then I got the error message above in my webbrowser, too. I switched to profOnnnos git version "defang" of pump.io and then it worked. 

Just wanted to thank @profOnno and point out that he is probably right, that there is something wrong in the git master.

Anyhow, pump.io still gives "undefined errors" in the browser, and looks pretty unstable, so I probably wont use it further, even though it looks like the right way to do it.
